src/test/controllers/users.controller.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+import sampleUser from '../fixtures/loggedInUserData';
+import '../../server/models/users.model';
+
+/*
+ * Test for users.controller.
+ */
+
+let assert = require('assert');
```

Comment:
This should be replaced to 

import assert from 'assert';

require is deprecated in es6.